src/homepage/howTo/custom-listeners.html

```diff
@@ -0,0 +1,237 @@
+$ cp -rf james-project/dockerfiles/run/guice/cassandra/destination/conf conf
+                </code></pre>
+
+                <p>Then create the keystore file in the conf/ directory with the default password <code>james72laBalle</code>
+                <pre><code>
+$ keytool -genkey -alias james -keyalg RSA -keystore conf/keystore
+                </code></pre>
+
+                <p>
+                    Second, modify listener.xml configuration file in conf/ directory to use only BigMessageListener
+                    by specifying its full class name. We only need to use this Listener in our example.
+                </p>
+                <pre><code>
+&lt;listeners&gt;
+    &lt;listener&gt;
+        &lt;class&gt;org.apache.james.examples.custom.listeners.BigMessageListener&lt;/class&gt;
+    &lt;/listener&gt;
+&lt;/listeners&gt;
+                </code></pre>
+
+                <p>Finally, starting a James Server by docker compose</p>
+                <p>Getting James docker-compose.yml</p>
+                <pre><code>
+$ wget https://raw.githubusercontent.com/apache/james-project/master/dockerfiles/run/docker-compose.yml
+                </code></pre>
+                <p>Add the following volumes for james service:</p>
+                <pre><code>
+volumes:
+  - $PWD/conf:/root/conf/
+                </code></pre>
+
+                <p>
+                    When you are using your listeners, place the jar containing your listeners into "extensions-jars" directory
+                    and add a volume for it.
+                </p>
+                <pre><code>
+volumes:
+  - $PWD/extensions-jars:/root/extensions-jars/
+                </code></pre>
+                <p>
+                    In this case, you should compile James project and put its jar of module <b>examples/custom-listeners</b> to "extensions-jars".
+                    The jar name can be sightly different by the time because its name carries James version, but you can recognize it easily.
+                </p>
+                <pre><code>
+$ mkdir extensions-jars
+$ cd james-project
+$ mvn clean install -DskipTests -T 1C
+$ cd ../
+$ cp james-project/examples/custom-listeners/target/custom-listeners-****.jar extension-jars/custom-listeners.jar
+                </code></pre>
+                <p>
+                    Check out the docker-compose.yml to get the host name of james service, currently it's <b>james.local</b>.
+                    So you, have to modify James domain list to use this <b>james.local</b> as James default domain:
+                </p>
+                <pre><code>
+&lt;domainlist&gt;
+    &lt;autodetect&gt;true&lt;/autodetect&gt;
+    &lt;autodetectIP&gt;true&lt;/autodetectIP&gt;
+    &lt;defaultDomain&gt;james.local&lt;/defaultDomain&gt;
+&lt;/domainlist&gt;
+                </code></pre>
+                <p>
+                    Now, you can able to start james by docker compose, and wait all docker containers up.
```

Comment:
Now, you can able to start james by docker compose, and wait all docker containers up.
=> 
Now, you are able to start james by docker compose, then wait for all docker containers to be up.